fast64_internal/f3d/f3d_gbi.py

```diff
@@ -4818,7 +4824,12 @@ class DPSetOtherMode(GbiMacro):
     mode1: list
 
     def to_binary(self, f3d, segments):
-        words = _SHIFTL(f3d.G_RDPSETOTHERMODE, 24, 8) | _SHIFTL(self.mode0, 0, 24), self.mode1
+        mode0 = mode1 = 0
+        for mode in self.mode0:
+            mode0 |= getattr(f3d, mode) if hasattr(f3d, str(mode)) else mode
```

Comment:
why is getattr given `mode` and `hasattr` is given `str(mode)` ?

`getattr` can also take a default if it simplifies things:
`getattr(f3d, mode, mode)`

Reply:
because I referenced SPSetOtherMode, should I update it too?